File: Better Deal.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int a,b,da,db;
        scanf("%d%d",&a,&b);
        da=(a*100)/100;
        db=(b*200)/100;
        if(100-da>200-db)
        printf("SECOND\n");
        else if(100-da<200-db)
        printf("FIRST\n");
        else
        printf("BOTH\n");
    }
}

File: Better Deal.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int a=sc.nextInt();
            int b=sc.nextInt();
            if((100-a)<(200-(2*b)))
            System.out.println("FIRST");
            else if((100-a)>(200-(2*b)))
            System.out.println("SECOND");
            else
            System.out.println("BOTH");
        }
    }
}